test/package/test_package_fx.py

```diff
@@ -17,19 +17,31 @@
     from common import PackageTestCase
 
 
+class CustomTracer(Tracer):
+    def is_leaf_module(self, m, module_qualified_name,) -> bool:
+        return super().is_leaf_module(m, module_qualified_name) or module_qualified_name == "MyModule"
+
+
+class MyModule(torch.nn.Module):
+    def forward(self, x):
+        return torch.relu(x)
+
+
+class SimpleTest(torch.nn.Module):
+    def forward(self, x):
+        return torch.relu(x + 3.0)
```

Comment:
I don't know why, but these are not pickleable when defined in local scope (also why are `MyModule` and `SimpleTest` being pickled in the first place?).